src/System.Net.Http/src/System/Net/Http/HttpContent.cs

```diff
@@ -191,7 +192,7 @@ public Task<string> ReadAsStringAsync()
                 Encoding encoding = null;
                 int bomLength = -1;
 
-                byte[] data = innerThis.GetDataBuffer(innerThis._bufferedContent);
+                byte[] data = innerThis._bufferedContent.ToArray();
```

Comment:
@stephentoub Changed it to `ToArray` because I was planning to make a separate PR for the `ReadAsStringAsync` method. It appears I've accidentally deleted that branch though, so I guess give me a few hours and I'll re-fix this method.